specification/ambulance-analytics.yaml

```diff
@@ -83,29 +86,35 @@ servers:
 tags:
   - name: Ambulance Analytics Api
 paths:
-
-  /x-dataset-id:
+  /dataset-id:
     post:
       summary: Post Ambulance Analytics data.
       operationId: postData
       description: |
-        Get a 200 response with an empty response body "{}"
-
+        Accepts and stores [FHIR](https://digital.nhs.uk/developer/guides-and-documentation/api-technologies-at-nhs-digital#fhir)
+        compatible payload. Successfull call will return 200 with an empty json body. Refer to Authorisation section of the description
+        for more details.
+        **Note**: Payload validation is done via MESH API.
```

Comment:
This endpoint doesn't validate posted payload and it will only acknowledge the receipt of the payload